### ISSUE TITLE
We failed to load the share invite inquiries for this monetary account

### ISSUE THREAD

**Issue by jbenitosantoni** [bunqCommunity/bunqDesktop#539]
When I open the app with my account already logged I always get this error message, also when I try to update the info on my account or make any movement.

We failed to load the share invite inquiries for this monetary account:

Route not found.

**Comment by LorenzoS92** [comment]
Same here. Every tab i press, i get the same error.

**Comment by Crecket** [comment]
Duplicate off #536

**Comment by JeroenEeuwes** [comment]
I still get this error with 0.9.9. Not as often in the previous release, but none-the-less. After a few errors I also get a "you tries too many times" error. See below.

<img width="433" alt="api-key-not-valid" src="https://user-images.githubusercontent.com/23123995/74632571-f4dba600-515f-11ea-9074-0fdb9607b3a1.png">

<img width="449" alt="too-many-times" src="https://user-images.githubusercontent.com/23123995/74632567-f1e0b580-515f-11ea-9075-89863edab1ee.png">

**Comment by firstred** [comment]
> I still get this error with 0.9.9. Not as often in the previous release, but none-the-less. After a few errors I also get a "you tries too many times" error. See below.
> 
> <img alt="api-key-not-valid" width="433" src="https://user-images.githubusercontent.com/23123995/74632571-f4dba600-515f-11ea-9074-0fdb9607b3a1.png">
> 
> <img alt="too-many-times" width="449" src="https://user-images.githubusercontent.com/23123995/74632567-f1e0b580-515f-11ea-9075-89863edab1ee.png">

Hi @JeroenEeuwes 

How old is your API key? I've read something about old API keys that are being invalidated due to backend changes, but I am not sure if that's the cause of these errors.
If renewing the API key does not work, could you please open a new issue? I think it's big enough of an issue to have a thorough look at again.

Changelog item that mentions the renewal: https://beta.doc.bunq.com/basics/changelog#april-8-2020

**Comment by JeroenEeuwes** [comment]
@firstred It was a new API key because it was a fresh install from scratch. I created a second new API key and so far I did not get the error again. So I don't really know how it happened with the first new API key. I'll update this issue when I get the same error with the second new API key.

**Comment by JeroenEeuwes** [comment]
@firstred Today I got the same error again. I left bunqDesktop open yesterday and put the PC in sleep mode. This morning bunqDesktop didn't get new transactions when I pressed on the refresh button. After I pressed it 3 times I got the error about too many tries again. There was no error before that (so nothing about expired sessions or whatever).
<img width="454" alt="invalid-code-too-many-times" src="https://user-images.githubusercontent.com/23123995/74720126-91bc4300-5235-11ea-8379-a2c43277fdb6.png">
